packages/web/modals/profile.tsx

```diff
@@ -77,17 +80,48 @@ export const ProfileModal: FunctionComponent<
     onOpen: onOpenFiatOnrampSelection,
     onClose: onCloseFiatOnrampSelection,
   } = useDisclosure();
+  const {
+    isOpen: isStargazePfpSelectOpen,
+    onClose: onCloseStargazePfpSelect,
+    onOpen: onOpenStargazePfpSelect,
+  } = useDisclosure();
 
   const wallet = accountStore.getWallet(chainId);
+  const address = wallet?.address ?? "";
+  const endpoint = "https://graphql.mainnet.stargaze-apis.com/graphql";
 
   const [hasCopied, setHasCopied] = useState(false);
   const [_state, copyToClipboard] = useCopyToClipboard();
   const [_isReady, _cancel, reset] = useTimeoutFn(
     () => setHasCopied(false),
     2000
   );
+  const [stargazeAddress, setStargazeAddress] = useState("");
+  const [stargazeNFTs, setStargazeNFTs] = useState<any[]>([]);
```

Comment:
Let's not use `any` here, can copy what you need from `Token` and `Collection` types from our schema in Galaxy